README.md

```diff
@@ -46,19 +46,17 @@ cart": {
 }
 ````
 
-4. Register the extension in `src/modules/index.ts` file and [disable the 'Instant checkout' module](https://github.com/Vendic/vsf-external-checkout/issues/11)
+4. Register the extension in `src/modules/client.ts` file and [disable the 'Instant checkout' module](https://github.com/Vendic/vsf-external-checkout/issues/11)
 ````js
-import { ExternalCheckout } from './external-checkout'
-// import { InstantCheckout } from './instant-checkout'
-
-export const registerModules: VueStorefrontModule[] = [
-   // other extensions
-  ExternalCheckout,
-  // InstantCheckout,
-]
+import { ExternalCheckout } from 'external-checkout'
```

Comment:
@cewald please restore previous module path value, should be
`import { ExternalCheckout } from './external-checkout'`
path `'external-checkout'` is incorrect and will cause error during installation process